src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();

```

Comment:
Don't use static methods or static fields in your solution. Instead, you can create an instance of the class where you need it.